Sources/XCUITestHelpers/XCUIDevice+UserInterfaceStyle.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+import XCTest
+
+public extension XCUIDevice {
```

Comment:
I thought about making it a `UIDevice` extension, but opted to keep the
logic on an `XCUI-` object to be consistent with the UI tests proxied
philosophy, where the application, and by extension the types exposed by
UIKit, are not accessed directly, but through the `XCUI-` objects and APIs.